core/agents/orchestrator.py

```diff
@@ -198,7 +199,7 @@ def create_agent(self, prev_response: Optional[AgentResponse]) -> BaseAgent:
             # Ask the Tech Lead to break down the initial project or feature into tasks and apply project template
             return TechLead(self.state_manager, self.ui, process_manager=self.process_manager)
 
-        if state.current_task and state.docs is None:
+        if state.current_task and state.docs is None and state.specification.complexity != Complexity.SIMPLE:
```

Comment:
this means we will not update `state.docs` so it will stay `None` which is different from `[]` which we have currently. Is that ok?